solr/modules/ltr/src/java/org/apache/solr/ltr/model/NeuralNetworkModel.java

```diff
@@ -341,9 +341,12 @@ protected void validate() throws ModelException {
   }
 
   @Override
-  public float score(float[] inputFeatures) {
-
-    float[] outputVec = inputFeatures;
+  public float score(Float[] inputFeatures) {
```

Comment:
is this necessary? what happens if we keep the wrapped version?
What happens if we change the function layer.calculateOutput instead?

Reply:
Do you want the neural network to manage null values?
In this way, I avoid introducing nulls in the network layers, if I change output from float to Float I need to also return a Float, and change other functions and tests..
Maybe you mean something different?